modules/filters.html

```diff
@@ -0,0 +1,182 @@
+      <form>
+        <fieldset>
+          <legend>Select your filters</legend>
+          <label>
+            <input type="range" name="blur"
+              value="0" min="0" max="1" step="0.1" />
+            blur()
+          </label>
+          <label>
+            <input type="range" name="brightness"
+              value="1" min="0" max="2" step="0.1" />
+            brightness()
+          </label>
+          <label>
+            <input type="range" name="contrast"
+              value="1" min="0" max="2" step="0.1" />
+            contrast()
+          </label>
+          <label>
+            <input type="range" name="dropShadow"
+              value="0" min="-1" max="1" step="0.1"  />
+            drop-shadow()
+          </label>
+          <label>
+            <input type="range" name="grayscale"
+              value="0" min="0"  max="1" step="0.1" />
+            grayscale()
+          </label>
+          <label>
+            <input type="range" name="hueRotate"
+              value="0" min="-1" max="1" step="0.1" />
+            hue-rotate()
+          </label>
+          <label>
+            <input type="range" name="invert"
+              value="0" min="0" max="1" step="0.1" />
+            invert()
+          </label>
+          <label>
+            <input type="range" name="opacity"
+              value="1" min="0" max="1" step="0.1" />
+            opacity()
+          </label>
+          <label>
+            <input type="range" name="saturate"
+              value="1" min="0" max="2" step="0.1" />
+            saturate()
+          </label>
+          <label>
+            <input type="range" name="sepia"
+              value="0" min="0" max="1" step="0.1" />
+            sepia()
+          </label>
+          <button type="reset">Reset</button>
+        </fieldset>
+      </form>
+    </article>
+    <script>
+      const image = document.querySelector("img");
+      const controls = document.querySelectorAll("input");
+      let resetValues = new Array();
+
+      for (control of controls) {
+        resetValues.push(control.value);
+        control.addEventListener( "change", () => {
+            changeCSS(); 
+          }, false );
+      }
+      document.querySelector("button").addEventListener( "click", () => {
+          setTimeout(function () {
+            changeCSS();
+          }, 50);
+        },
+        false
+      );
+
+      function changeCSS() {
+        image.setAttribute("style", "filter: " +
+            blur() +
+            brightness() +
+            contrast() +
+            dropShadow() +
+            grayscale() +
+            hueRotate() +
+            invert() +
+            opacity() +
+            saturate() +
+            sepia()
+        );
+      }
+      /* blur()*/
+      function blur() {
+        let blurValue = document.getElementsByName("blur")[0];
+        return "blur(" + blurValue.value + "rem) ";
+      }
```

Comment:
I removed the comment as it doesn't bring anything: it is merely the function's name that appears on the following line. I'm open to another comment, though. (The others are the recommendations that came out of the Modernize JS project)
```suggestion

      function blur() {
        const blurValue = document.getElementsByName("blur")[0];
        return `blur(${blurValue.value}rem) `;
      }
```